src/views/Layout.vue

```diff
@@ -142,6 +142,11 @@ export default {
         }
       },
     },
+    '$route.path' (to) {
+      if (to === '/namespaces') {
```

Comment:
This if is not necessary, set it everytime. And if its special case it will be overwritten